test/functional/rpc_getblockstats.py

```diff
@@ -40,40 +40,47 @@ def get_stats(self):
         return [self.nodes[0].getblockstats(hash_or_height=self.start_height + i) for i in range(self.max_stat_pos+1)]
 
     def generate_test_data(self, filename):
+        node = self.nodes[0]
```

Comment:
I suggest you don't make this change to our patch set. If this file changes in master, then a change like this will make rebasing more difficult.